oracle/src/sponge.rs

```diff
@@ -142,7 +142,9 @@ where
         self.last_squeezed = vec![];
         for g in g.iter() {
             if g.infinity {
-                panic!("sponge got zero curve point");
+                // absorb a fake point (0, 0)
```

Comment:
In the future -- I think we should guarantee that commitments are not point at infinity by blinding them. Fine for now though